Makefile

```diff
@@ -44,8 +44,19 @@ OBJCOPY = $(CROSS_COMPILE)objcopy
 SIZE    = $(CROSS_COMPILE)size
 GDB     = $(CROSS_COMPILE)gdb
 
+# Flasher utility options
 NRFUTIL = adafruit-nrfutil
 NRFJPROG = nrfjprog
+FLASHER ?=
+
+# default to nrf
+ifeq ($(FLASHER),)
+  FLASHER = nrf
+else ifeq ($(FLASHER),pyocd)
+  PYOCD ?= pyocd

```

Comment:
we could simplify the target rule by define FLASH_CMD function that take hex file as input and expand to appropriate flash command.  Much cleaner than defining each target for each flasher. However If you are not familiar with makefile function define, feel free to skip it, I will do it later on.